route/v1/types.go

```diff
@@ -26,6 +26,13 @@ import (
 // return information to clients about the names and states of the route under each router.
 // If a client chooses a duplicate name, for instance, the route status conditions are used
 // to indicate the route cannot be chosen.
+//
+// To enable HTTP/2 ALPN on a route it requires a custom certificate. This
+// prevents connection coalescing by clients, notably web browsers. We do not
```

Comment:
@frobware do we need to clarify that the custom cert can not be a wildcard cert?

Reply:
Done in https://github.com/openshift/api/pull/654/commits/f48795ed59406cc07964980c43b53c46a3d2ffee